__tests__/app.test.js

```diff
@@ -72,6 +72,7 @@ describe("API's", () => {
               votes: 100,
               article_img_url:
                 "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+              comment_count: 11,
```

Comment:
Ideally, TDD should be _Open to expansion, but Closed to modification_
So in an ideal world this new functionality should have been tested using a new test block, rather than going back and changing the test 💡